### PR TITLE
Submitting Squiid version 1.0.0

### PR DESCRIPTION
**Pull request by TabulateJarl8**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113263&drop=dogfoodAlpha

### CODE REVIEW

**Comment by TabulateJarl8** [comment]
Could someone take a look at this? It installed without interaction (except for the UAC prompt, maybe that's the problem) on my local computer so I'm not sure what else to check

**Comment by stephengillie** [comment]
Hi @TabulateJarl8,

Automatic Validation ended with:
image

Is there an installer switch to select the install mode and bypass this prompt?

**Comment by TabulateJarl8** [comment]
I believe this will fix the issue:
```yaml
InstallerSwitches:
    Custom: /ALLUSERS
```

File: manifests/i/ImaginaryInfinity/Squiid/1.0.0/ImaginaryInfinity.Squiid.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ImaginaryInfinity.Squiid
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: ImaginaryInfinity
+PackageName: Squiid
```
Comment (stephengillie) [inline manifests/i/ImaginaryInfinity/Squiid/1.0.0/ImaginaryInfinity.Squiid.locale.en-US.yaml:8]:
```suggestion
PackageName: Squiid
PackageUrl: https://gitlab.com/ImaginaryInfinity/squiid-calculator/squiid/
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**